src/geometry/CTiglPatchShell.cpp

```diff
@@ -158,24 +158,29 @@ TopoDS_Shell MakeShells(TopoDS_Shape const& shell, const Standard_Real tol)
         StdFail_NotDone::Raise("Loft is not build");
```

Comment:
If we are switching to TiGL exceptions, we might also replace this exception here.